File: src/main/java/nu/peg/slack/pt/api/slack/model/ErrorMessage.java
package nu.peg.slack.pt.api.slack.model;

import java.util.Arrays;

public class ErrorMessage extends SlackMessage {

    public ErrorMessage(String error) {
        setAttachments(Arrays.asList(
                new ErrorAttachment("#ff0000", String.format("*Error*: %s", error), Arrays.asList("text"))
        ));
    }
}

File: src/main/kotlin/nu/peg/slack/pt/api/slack/model/ErrorMessage.kt
package nu.peg.slack.pt.api.slack.model

import java.util.Arrays

class ErrorMessage(error: String) : SlackMessage() {

    init {
        setAttachments(Arrays.asList<T>(
                ErrorAttachment("#ff0000", String.format("*Error*: %s", error), Arrays.asList<T>("text"))
        ))
    }
}
